blueman/Functions.py

```diff
@@ -49,13 +47,11 @@
 from gi.repository import Gtk
 from gi.repository import Gdk
 from gi.repository import GdkPixbuf
-from gi.repository import GLib
 from gi.repository import Gio
 
-
 __all__ = ["check_bluetooth_status", "launch", "setup_icon_path", "adapter_path_to_name", "e_", "bmexit",
-           "format_bytes", "create_menuitem", "get_lockfile", "get_pid", "is_running", "check_single_instance", "kill",
-           "have", "set_proc_title", "create_logger", "create_parser", "open_rfcomm", "get_local_interfaces"]
+           "format_bytes", "create_menuitem", "is_running", "kill", "have", "set_proc_title", "create_logger",
```

Comment:
We could move `is_running` and `kill` into NetConf.

Reply:
That is a good idea, I'll push a new commit.